zephyr.py

```diff
@@ -29,7 +35,6 @@ def __new__(cls):
 
     def __del__(self):
         _z.cancelSubs()
-        super(Subscriptions, self).__del__()
```

Comment:
Hm, I removed this based on the last answer from http://stackoverflow.com/questions/36722390/python-3-super-del, but judging by https://docs.python.org/3.3/reference/datamodel.html#object.__del__, it looks like we really are supposed to call it. (Not that not calling it has obviously broken yet.)